src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/nomis/Address.kt

```diff
@@ -3,7 +3,29 @@ package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis
 import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.Address as IntegrationAPIAddress
 
 data class Address(
-  val postalCode: String,
+  val addressType: String?,
+  val country: String?,
+  val county: String?,
+  val endDate: String?,
+  val flat: String?,
+  val locality: String?,
+  val postalCode: String?,
+  val premise: String?,
+  val startDate: String?,
+  val street: String?,
+  val town: String?,
 ) {
-  fun toAddress() = IntegrationAPIAddress(postcode = this.postalCode)
+  fun toAddress() = IntegrationAPIAddress(
+    country = this.country,
+    county = this.county,
+    endDate = this.endDate,
+    locality = this.locality,
+    name = this.premise,
+    number = this.flat,
+    postcode = this.postalCode,
+    startDate = this.startDate,
+    street = this.street,
+    town = this.town,
+    type = this.addressType,
```

Comment:
I think this should be restructured and refactored:
flat = this.flat
premise = this.premise
name = "Something with this?"
streetAddress = this.street
addressLocality = this.locality
addressTown = this.town
addressRegion = this.county
addressCountry = this.country
postalCode = this.postalCode
startDate = this.startDate
endDate = this.endDate
additionalType = this.addressType